core/src/main/scala/org/bitcoins/core/node/NodeApi.scala

```diff
@@ -0,0 +1,29 @@
+package org.bitcoins.core.node
+
+import org.bitcoins.core.crypto.DoubleSha256Digest
+import org.bitcoins.core.util.FutureUtil
+
+import scala.concurrent.Future
+
+/**
+  * Represent a bitcoin-s node API
+  */
+trait NodeApi {
+
+  /**
+    * Request the underlying node to download the given blocks from its peers and feed the blocks to [[org.bitcoins.node.NodeCallbacks]].
+    */
+  def requestBlocks(blockHashes: Vector[DoubleSha256Digest]): Future[Unit]
```

Comment:
nit: This name makes it sound like the user is requesting blocks that the `NodeApi` has, maybe rename to `downloadBlocks`?